docs/kalite-theme/breadcrumbs.html

```diff
@@ -13,11 +13,11 @@
             <a href="https://bitbucket.org/{{ bitbucket_user }}/{{ bitbucket_repo }}/src/{{ bitbucket_version}}{{ conf_py_path }}{{ pagename }}{{ source_suffix }}" class="fa fa-bitbucket"> Edit on Bitbucket</a>
           {% elif show_source and source_url_prefix %}
             <a href="{{ source_url_prefix }}{{ pagename }}{{ source_suffix }}">View page source</a>
-          {% elif show_source and has_source and sourcename %}
+          {% elif "" and show_source and has_source and sourcename %}
```

Comment:
I'd say either remove these branches entirely or update this LOC: https://github.com/learningequality/ka-lite/issues/4348#issuecomment-182539950